lib/hsa/mcwamp_hsa.cpp

```diff
@@ -4970,14 +4969,14 @@ HSACopy::waitComplete() {
     // Wait on completion signal until the async copy is finishedS
     if (DBFLAG(DB_WAIT)) {
         hsa_signal_value_t v = -1000;
-        if (signal.handle) {
-            hsa_signal_load_acquire(signal);
+        if (_signal.handle) {
+            hsa_signal_load_acquire(_signal);
```

Comment:
Unless I am mistaken, this is deprecated, with `hsa_signal_load_scacquire` being the current / preferred form.